src/languages/lsl.js

```diff
@@ -0,0 +1,133 @@
+/*
+    Language: Linden Scripting Language
+    Description: The Linden Scripting Language is used in Second Life by Linden Labs.
+    Author: Builder's Brewery <buildersbrewery@gmail.com>
+    Category: common, scripting
```

Comment:
The category 'common' is special as it controls what goes into the default hosted build which we don't want to become too big. Also, LSL is hardly common by any definition of the word :-)